docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+services:
```

Comment:
Let's rename this as `redis-docker-compose.yml` so that it's clear that this should only be used for redis deployments.